src/Common/tests/System/Net/TestWebProxies.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Test.Common
+{
+    internal sealed class UseSpecifiedUriWebProxy : IWebProxy
```

Comment:
Nothing changed in these test types; I simply separated them out into their own file.